Source/WebKit/Scripts/generate-serializers.py

```diff
@@ -329,12 +329,25 @@ def generate_impl(serialized_types, serialized_enums, headers):
                 result.append('    } else')
                 result.append('        ' + sanitize_string_for_variable_name(member.name) + ' = std::optional<' + member.type + '> { ' + member.type + ' { } };')
             else:
-                result.append('    decoder >> ' + sanitize_string_for_variable_name(member.name) + ';')
-                result.append('    if (!' + sanitize_string_for_variable_name(member.name) + ')')
-                result.append('        return std::nullopt;')
-                if 'ReturnEarlyIfTrue' in member.attributes:
-                    result.append('    if (*' + sanitize_string_for_variable_name(member.name) + ')')
-                    result.append('        return { ' + type.namespace_and_name() + ' { } };')
+                r = re.compile("SecureCodingAllowed=\[(.*)\]")
```

Comment:
When we add new things to this script, I like to add to Source/WebKit/Scripts/webkit/tests/TestSerializedType.serialization.in then run "make -C Source/WebKit/Scripts/webkit/tests" so we have a "test" of what the generated code is supposed to look like.